.README/rules/check-tag-names.md

```diff
@@ -75,10 +75,28 @@ version
 yields
 ```
 
+#### Options
+
+##### `definedTags`
+
+Use an array of `definedTags` strings to configure additional, allowed JSDoc tags.
+The format is as follows:
+
+```json
+{
+  "definedTags": ["define", "record"]
+}
+```
+
+##### `preferredTagsDefined`
+
+If this boolean is set to `true`, the tags indicated as replacements in `settings.jsdoc.tagNamePreference` will automatically be considered as defined.
```

Comment:
I don't think this needs to be an option. It does not make sense to set this to false.

Reply:
Good point, and I think I can also remove `preferredTypesDefined` which was added earlier for the same reason. Do you want the removal in this same PR?

Reply:
I like them to be in separate prs.  How about creating a new branch `11.0.0` and make it the target branch of these separate prs, and then when all things are done merge them together?

Reply:
Ok, sure.

Reply:
So I will remove `preferredTagsDefined` in this PR (i.e., avoid adding it at all) and remove `preferredTypesDefined` in the other.

Reply:
I've removed `preferredTagsDefined` from this PR and added #304 to remove `preferredTypesDefined`.